data/download_data.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import json
+from pprint import pprint
+import logging
+
+logger = logging.getLogger(__name__)
+
+class download_metadata():
```

Comment:
Classes should generally be CamelCase, not lower case, but also, this can just be a function instead, that would be a bit simpler for something that just downloads the metadata.

Reply:
sure, I was thinking if we need to expand on the meta data download for different `PV` site farms from the `api`, we can get back to using classes i guess!

Reply:
Okay, yeah, I think we could just add functions for downloading for different ones, or arguments for it to, it removes some boiler plate stuff.

Reply:
yes!